workflow/signal/signal.go

```diff
@@ -49,6 +53,8 @@ func ExecPodContainerAndGetOutput(ctx context.Context, restConfig *rest.Config,
 	if err != nil {
 		return err
 	}
+	ctx, cancel := context.WithTimeout(ctx, spdyTimeout)
```

Comment:
Make spdyTimeout a local variable to this function, rather than a global please.

Reply:
> Make spdyTimeout a local variable to this function, rather than a global please.

Thanks, i will fix it.